.github/workflows/indexnow.yaml

```
@@ -0,0 +1,21 @@
+name: Notify IndexNow on Deploy
+on:
+  deployment_status:
```

actionlint (1.7.11)

[error] "types" cannot be specified for "deployment_status" Webhook event

(events)